### DISCUSSION TITLE
Change pantry name?

### DISCUSSION THREAD

**Discussion by bklik** [imRohan/Pantry#142] [Q&A] [answered]
Is it possible to change your pantry name after naming it?

**Comment by imRohan** [comment, marked as answer]
@bklik Yes via the API but not through the dashboard UI.

Send a `PUT` request to `apiv1/pantry/YOUR_PANTRY_ID` with a JSON encoded body including the `name` attribute. Here's an example:

```
curl --location --request PUT 'https://getpantry.cloud/apiv1/pantry/YOUR_PANTRY_ID' \
--header 'Content-Type: application/json' \
--data-raw '{
	"name": "New Name Goes Here"
}'
```

Unfortunately, users are unable to change the name of a Pantry at this time from the dashboard UI. However, I am not opposed to the idea as changing the name would be very useful for users.
